packages/gatsby/src/utils/webpack.config.js

```diff
@@ -643,71 +632,79 @@ module.exports = async (
   }
 
   if (stage === `build-html` || stage === `develop-html`) {
+    // removes node internals from bundle
+    // https://webpack.js.org/configuration/externals/#externalspresets
+    config.externalsPresets = {
+      node: true,
+    }
+
```

Comment:
OMG, that's so nice that this is explicit and targetting `node` no longer implies it (in terms of being able of wrapping `fs` etc for conditional page builds)

Reply:
Scratch above - we still actually will need to mark those as externals :(